service/application/src/main/java/com/solace/maas/ep/event/management/agent/config/SolaceConfiguration.java

```diff
@@ -29,6 +30,7 @@
 @Slf4j
 @ExcludeFromJacocoGeneratedReport
 @Configuration
+@Profile("!TEST")
 @ConditionalOnProperty(name = "event-portal.gateway.messaging.standalone", havingValue = "false")
 public class SolaceConfiguration {
```

Comment:
We don't want this class present when we are testing (e.g. we don't want the real Beans)